python/runtime/mlir/py_register_dialects.cpp

```diff
@@ -143,21 +115,10 @@ void registerQuakeDialectAndTypes(py::module &m) {
       });
 }
 
-void registerCCDialectAndTypes(py::module &m) {
+void registerCCTypes(py::module &m) {
```

Comment:
```suggestion
static void registerCCTypes(py::module &m) {
```
Assuming that's ok, should move these 2 out of the namespace as well.

Reply:
Yeah, it makes sense to remove the namespace entirely in this `.cpp` file.